src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.html

```diff
@@ -16,6 +16,15 @@
   class="kuiViewContent kuiViewContent--constrainedWidth"
   data-test-subj="dashboardLandingPage"
 >
+  <div class="kuiViewContentItem kuiVerticalRhythm" ng-if="listingController.showLimitError">
+    <div class="kuiInfoPanel kuiInfoPanel--warning">
+      <div class="kuiInfoPanelBody">
+        <div class="kuiInfoPanelBody__message">
+          You have {{ listingController.totalItems }} dashboards, but your "listingLimit" setting prevents the table below from displaying more than {{ listingController.listingLimit }}. You can change this setting under Management &gt; Advanced Settings.
```

Comment:
I just realized we should probably just link directly to the Advanced Settings page. Can we change the last sentence to:

```html
You can change this setting under <a kbn-href="#/management/kibana/settings" class="kuiLink">Advanced Settings</a>.
```